src/main/java/com/ghilly/service/CountryServiceRest.java

```diff
@@ -66,4 +69,11 @@ private void exists(int countryId) {
             throw new IdIsNotFoundException("The country with this ID " + countryId + " is not found.");
```

Comment:
maybe it will be better to rename to IdNotFoundException

Reply:
Ok